index.html

```diff
@@ -809,18 +816,6 @@ <h3 id="notifications">
           non-secure contexts=] flag set.
         </p>
       </section>
-      <section>
-        <h3 id="midi">
-          Midi
-        </h3>
-        <p>
-          The <dfn>midi</dfn> enum value identifies the [[[webmidi]]] [=powerful feature=].
-        </p>
-        <p class="issue">
```

Comment:
Do we want to keep these?

Reply:
nah, it's fine to drop... will eventually delete them all anyway.